app/latigo/time_series_api/misc.py

```diff
@@ -76,7 +76,7 @@ def prediction_data_naming_convention(
     if not tag_name:
         tag_name = f"{facility}.{missing_tag_name}"
     if not model_name:
-        raise Exception(f"'model_name' can not be empty")
+        raise Exception(f"'{model_name}' can not be empty")
```

Comment:
what is the idea behind this change?
I suppose it should be 
```suggestion
        raise Exception("'model_name' can not be empty")
```